### PR TITLE
ogl-runtime: Check for ppc64 before ppc

### PR DESCRIPTION
**Pull request by kraj**
since compiler for ppc64 defines both ppc and ppc64 its better to
check for ppc64 first

Signed-off-by: Khem Raj <raj.khem@gmail.com>

### CODE REVIEW

**Comment by kraj** [comment]
@shr-project  I have pushed couple of gcc12 needed fixes to this pull as well. Hope thats ok

**Comment by shr-project** [comment]
All 3 are now in master-next, sorry for delay.

File: recipes-connectivity/libconnman-qt/libconnman-qt5/0001-Add-missing-declarations-for-operator-overloads.patch [hunk 1]
```diff
@@ -17,14 +17,12 @@ Signed-off-by: Khem Raj <raj.khem@gmail.com>
...
 +++ b/libconnman-qt/marshalutils.cpp
-@@ -36,6 +36,11 @@
+@@ -36,6 +36,9 @@
  
  #include "marshalutils.h"
  
-+#if defined(__clang__) &&  __clang_major__ >= 11
 +Q_DBUS_EXPORT const QDBusArgument &operator>>(const QDBusArgument &a, RouteStructure &v);
 +Q_DBUS_EXPORT QDBusArgument &operator<<(QDBusArgument &a, const RouteStructure &v);
-+#endif
```
Comment (shr-project) [inline recipes-connectivity/libconnman-qt/libconnman-qt5/0001-Add-missing-declarations-for-operator-overloads.patch:27]: This one doesn't seem to be compatible with gcc-11 currently used in oe-core/master

http://errors.yoctoproject.org/Errors/Details/655056/

**Comment by shr-project** [comment]
ogl-runtime and maliit-* changes merged, thanks. For connman please create new PR with updated patch.

File: recipes-connectivity/libconnman-qt/libconnman-qt5/0001-Add-missing-declarations-for-operator-overloads.patch (hunk 1 above)
Comment (kraj) [reply recipes-connectivity/libconnman-qt/libconnman-qt5/0001-Add-missing-declarations-for-operator-overloads.patch:27]: now gcc 12 is default in core/master

**Pull request closed without merging**